src/globus_sdk/experimental/globus_app/globus_app.py

```diff
@@ -126,17 +126,17 @@ class GlobusAppConfig:
 
 class GlobusApp(metaclass=abc.ABCMeta):
     """
-    ``GlobusApp` is an abstract base class providing an interface for simplifying
+    ``GlobusApp`` is an abstract base class providing an interface for simplifying
     authentication with Globus services.
 
     A ``GlobusApp`` manages scope requirements across multiple resource servers,
     runs login flows through its ``LoginFlowManager``, handles token storage and
     validation through its ``ValidatingTokenStorage``, and provides up-to-date
     authorizers from its ``AuthorizerFactory`` through ``get_authorizer``.
 
-    In the near future, a ``GlobusApp`` will be accepted as an initialization parameter
-    to any Globus service client, automatically handling the client's default
-    scope requirements and providing the client an authorizer.
+    A ``GlobusApp`` is accepted as an initialization parameter to any SDK-hosted Globus
```

Comment:
Is "hosted" a bit of an odd fit here? Maybe:

```suggestion
    A ``GlobusApp`` is accepted as an initialization parameter to any SDK-provided Globus
```